src/core/biblio-db.js

```diff
@@ -173,8 +171,8 @@ export const biblioDB = {
       }, aliasesAndRefs);
     const promisesToAdd = Object.keys(aliasesAndRefs)
       .map(type => {
-        return Array.from(aliasesAndRefs[type]).map(details =>
-          await this.add(type, details)
+        return Array.from(aliasesAndRefs[type]).map(
+          async details => await this.add(type, details)
```

Comment:
This change is redundant because the return type is still Promise, and it's expected.